components/brave_wallet/common/hash_utils_unittest.cc

```diff
@@ -79,4 +79,30 @@ TEST(HashUtilsUnitTest, Hash160) {
             "b472a266d0bd89c13706a4132ccfb16f7c3b9fcb");
 }
 
+TEST(HashUtilsUnitTest, HmacSha512) {
+  // Empty vectors test.
+  EXPECT_EQ(HexEncodeLower(HmacSha512({}, {})),
```

Comment:
nit: how were the empty and large vectors generated?

Reply:
This are not test vectors from public source. Just captures function behavior on edge cases

Reply:
Ahh gotcha, yeah no worries. I saw that you included the other ones from public which should be good enough here.